Sorts/QuickSort.ts

```diff
@@ -39,10 +39,10 @@
  * @param {Array<number>} array
```

Comment:
You missed this.

Reply:
Please refresh, had updated in previous commit.